v1-migration.md

```diff
@@ -117,7 +115,7 @@ The eCom v2 API is available at ```ecomm/v2/payments```:
 
 ## Documentation
 
-The eCom v2 API documentation is available here: https://github.com/vippsas/vipps-ecom-api
+The eCom v2 API documentation is available here: <https://github.com/vippsas/vipps-ecom-api>
```

Comment:
What do the brackets do?

Reply:
The brackets just get rid of a warning against "bare urls".